app/src/main/java/org/mozilla/fenix/compose/ListItemTabLargePlaceholder.kt

```diff
@@ -0,0 +1,81 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.compose
+
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.isSystemInDarkTheme
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Card
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import mozilla.components.ui.colors.PhotonColors
+import org.mozilla.fenix.theme.FirefoxTheme
+
+/**
+ * Placeholder of a [ListItemTabLarge] with the same dimensions but only a centered text.
+ * Has the following structure:
+ * ```
+ * ---------------------------------------------
+ * |                                           |
+ * |                                           |
+ * | Placeholder text                          |
+ * |                                           |
+ * |                                           |
+ * ---------------------------------------------
+ * ```
+ */
+@Composable
+fun ListItemTabLargePlaceholder(
+    text: String,
+    onClick: () -> Unit = { }
+) {
+    Card(
+        elevation = 6.dp,
+        shape = RoundedCornerShape(8.dp),
+        modifier = Modifier
+            .size(328.dp, 116.dp)
+            .clickable { onClick() },
+        backgroundColor = FirefoxTheme.colors.surface,
+    ) {
+        Column(
+            modifier = Modifier
+                .fillMaxSize()
+                .padding(16.dp),
+            verticalArrangement = Arrangement.Center
+        ) {
+            Text(
+                textAlign = TextAlign.Center,
```

Comment:
I do generally like specifying the param names because it makes it easier in terms of code review to know how the arguments matches up, but I also find myself reordering them to stick with the order that `Text` defines. For instance, `text` would usually appear first follow by `color`, etc.

Reply:
Fair point. Thanks!